src/tmp.jl

```diff
@@ -240,26 +240,52 @@ function tmp_create_union_tables(connection)
     # These are equivalent to the partitions in time-resolution.jl
     # But computed in a more general context to be used by variables as well
 
+    # Incoming flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_in_flows AS
+        SELECT DISTINCT to_asset as asset, year, rep_period, time_block_start, time_block_end
+        FROM flow_time_resolution
+        ",
+    )
+
+    # Outgoing flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_out_flows AS
+        SELECT DISTINCT from_asset as asset, year, rep_period, time_block_start, time_block_end
+        FROM flow_time_resolution
+        ",
+    )
+
+    # Union of all assets and outgoing flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_assets_and_out_flows AS
+        SELECT DISTINCT asset, year, rep_period, time_block_start, time_block_end
+        FROM asset_time_resolution
+        UNION
+        FROM t_union_out_flows
```

Comment:
Sorry I'm also learning SQL a bit in the process: is it fine to neglect `SELECT` after `UNION` :)

Reply:
Not for all SQL languages, but DuckDB allows writing `FROM table` without the `SELECT`. It means `SELECT * FROM table`. We can be explicit, if preferred